src/dex/algebra/algebra.ts

```diff
@@ -176,13 +176,19 @@ export class Algebra extends SimpleExchange implements IDex<AlgebraData> {
 
     try {
       this.logger.info(
-        `${logPrefix} delete pool from not existing set: ${poolKey}`,
+        `${logPrefix} delete pool from not existing set=${this.notExistingPoolSetKey}; key=${poolKey}`,
       );
       // delete pool record from set
-      await this.dexHelper.cache.zrem(this.notExistingPoolSetKey, [poolKey]);
+      const result = await this.dexHelper.cache.zrem(
+        this.notExistingPoolSetKey,
+        [poolKey],
+      );
+      this.logger.info(
+        `${logPrefix} delete pool from not existing set=${this.notExistingPoolSetKey}; key=${poolKey}; result: ${result}`,
+      );
```

Comment:
Does it make sense to `log` it, if we will log it anyway in case of error?

Reply:
Here it's mostly to log the `result`, if it's > 0, means we actually deleted smth, if it's 0, means there wasn't any pool pair by that key

Error is thrown when we try to run zrem against non-sets